ddln_imaginator.proto

```diff
@@ -185,7 +186,45 @@ message WorldInfo {
     optional LatLongAlt home_geo = 4;
 }
 
+// Imaginator info message which contains imaginator specific information
+// This information is world agnostic and typically generated upon initialization
+message ImaginatorInfo {
+
```

Comment:
it'd be nice if even the first version of this message included things like:
- Unigine version
- git commit hash if possible

Reply:
This is another task, the first 9 fields will be reserved for such information.